stream-chat-android-ui-common/src/test/kotlin/com/getstream/sdk/chat/Mother.kt

```diff
@@ -197,10 +197,11 @@ internal fun createAttachment(
     fallback: String? = randomString(),
     uploadFile: File? = null,
     uploadState: Attachment.UploadState? = null,
-    extraData: MutableMap<String, Any> = mutableMapOf()
+    extraData: MutableMap<String, Any> = mutableMapOf(),
+    authorLink: String?
 ): Attachment = Attachment(
-    authorName, titleLink, thumbUrl, imageUrl, assetUrl, ogUrl, mimeType,
-    fileSize, title, text, type, image, url, name, fallback, uploadFile, uploadState, extraData
+    authorName, authorLink, titleLink, thumbUrl, imageUrl, assetUrl, ogUrl, mimeType,
```

Comment:
It would be nice to start using named parameters here so we don't accidentally pass them in wrong order